sql/2021/pwa/sw_events.sql

```diff
@@ -0,0 +1,28 @@
+#standardSQL
+# SW events - based on 2019/14_03.sql
+#
+# TODO - CONVERT TO USE CUSTOM METRICS
+#
+SELECT
+  client,
+  event,
+  COUNT(DISTINCT page) AS freq,
+  total,
+  COUNT(DISTINCT page) / total AS pct
+FROM
+  `httparchive.almanac.service_workers`
+JOIN
+  (SELECT client, COUNT(DISTINCT page) AS total FROM `httparchive.almanac.service_workers` WHERE date = '2021-07-01' GROUP BY client)
+USING (client),
+  UNNEST(ARRAY_CONCAT(
```

Comment:
This one is unchecked, so I assume I can just update it with the new methods defined in [pwa.js](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/pwa.js).